src/cmd/promremotebench/write_test.go

```diff
@@ -40,83 +38,72 @@ import (
 )
 
 func TestRemoteWrite(t *testing.T) {
-	tests := []struct {
```

Comment:
Hm, I kinda think it's better to have these as separate tests like it was before, rather than one big test that tests multiple different cases at the bottom - don't you think?